### ISSUE TITLE
Inconsistency of data normalization naming with sklearn API 

### ISSUE THREAD

**Issue by sdimi** [mwaskom/seaborn#1423]
Minor issue but might lead to confusion.

Seaborn's clustermap calls the rescaling of features [0, 1] or `(x-min(x)) / (max(x)-min(x))` as **standard_scale**: https://github.com/mwaskom/seaborn/blob/master/seaborn/matrix.py#L895

while sklearn calls it **minmax_scale**. To make things worse, sklearn calls **StandardScaler** this normalization:` (x-mean(x))/std(x)`.

Maybe would be nice to change seaborn's standard_scale to minmax_scale.

**Comment by mwaskom** [comment]
Normalization transformations have an incredibly inconsistent naming convention across disciplines and subfields. I assume @olgabot named those terms with the way they are conventionally used in computational biology, where the cluster heatmaps are very common. Using scikit-learn terminology would also have been a good idea, given the popularity of that package, but as seaborn doesn't depend on/interface with scikit-learn directly, it makes the inconsistency less jarring. My preference then would be not to force people to change their code just for the sake of moving from one arbitrary terminological reference frame to another.
